src/State.cpp

```diff
@@ -49,7 +49,12 @@ void State::Draw()
 
 						lastVertexDescriptor = currentVertexDescriptor;
 						lastPixelDescriptor = currentPixelDescriptor;
+					}
+
+					auto shadowState = RE::BSGraphics::RendererShadowState::GetSingleton();
+					auto cubeMapRenderTarget = !REL::Module::IsVR() ? shadowState->GetRuntimeData().cubeMapRenderTarget : shadowState->GetVRRuntimeData().cubeMapRenderTarget;
```

Comment:
Change to `GET_INSTANCE_MEMBER(cubeMapRenderTarget, shadowState)`

Reply:
done